src/LinuxTracing/PerfEventReaders.cpp

```diff
@@ -295,128 +296,55 @@ MmapPerfEvent ConsumeMmapPerfEvent(PerfEventRingBuffer* ring_buffer,
 
 StackSamplePerfEvent ConsumeStackSamplePerfEvent(PerfEventRingBuffer* ring_buffer,
                                                  const perf_event_header& header) {
-  // We expect the following layout of the perf event:
-  //  struct {
-  //    struct perf_event_header header;
```

Comment:
Are you sure about erasing this comment? Why not just erasing the last part of the computation of offsets?

Reply:
I don't think they are needed anymore. At first the functions were really messy and you needed to see the struct to understand what was going on but now you should be able to understand what's happening just by looking at the flags. The comment doesn't add much info and it clutters the view. Besides we already have PerfEventRecord as an actual struct in the code, if you want you can just look at the flags and then look at the struct and you'll understand perfectly what's being populated.